### ISSUE TITLE
Language server cannot start bsb

### ISSUE THREAD

**Issue by ser1us** [jaredly/reason-language-server#172]
Hi there

I just installed the extension on my second machine and it immediately fails with this error message:

```
C:\Users\chris\Development\reason-ml\my-new-project\node_modules\.bin\bsb' is not recognized as an internal or external command,
operable program or batch file.
```

And here is the message from the log file

```
Sending response {"id": 1, "jsonrpc": "2.0", "error": {"code": -32603, "message": "Could not run bsb (ran C:\\Users\\chris\\Development\\reason-ml\\my-new-project\\node_modules\\.bin\\bsb -version). Output: "}}
```

I don't know, it looks to me that the command is missing a file extension (`bsb.cmd`) instead of `bsb`? It works on my primary machine but I cannot see the command which is being executed in the logs there.

Thanks a lot for the extension!

Cheers,
Christoph

OS: Windows 10
VSCode: 1.28.2

**Comment by phated** [comment]
My friend ran into a similar issue (but on MacOS) and it was a permission issue. He had to start vscode with the CLI: `code .`

You might need to install the CLI using the command pallette in vscode.

**Comment by jaredly** [comment]
Ah yes, I should have asked that first. `WSL` is not supported -- we recommend installing node, bs-plartform, etc. exclusively through the windows side of things.

**Comment by ser1us** [comment]
TL;DR: The issue seems to be related to the cmd PATHEXT environment variable and/or some related registry settings. I double checked it on a new local account and everything worked just fine.

No no, the problem is definitely related to powershell/cmd :) I tried it in WSL (where it worked btw) just for fun. 

Anyway, I did some further testing and added a new local user account and installed VS Code as well as your extension. And it just worked. Back in my account I tried to execute the command using a relative path: `.\node_modules\.bin\bsb` and to my surprise this worked too!

In the end I deleted my user account, recreated it and now it's working just fine. 

Thanks for trying to help me :) Much appreciated.

Cheers

**Comment by satvikpendem** [comment]
I also get the same error, also installed ReasonML, VSCode, and this extension via Windows. The command `bsb.cmd -version` works but not `bsb -version`. Is there a way to just check for Windows and use the `.cmd` versions? This seems to be the simplest solution to this problem.
